Add repeat-part feature background size extends

pweibert opened the pull request:
Add a feature that would allow not only to scale the specified parts of the texture but also to repeat them. This feature can be turned on and off by a corresponding setter.

samskivert commented:
This PR contains a bunch of unnecessary code reformatting. Can you only include the code that's actually changing, and match the style of the existing code?

pweibert commented:
> This PR contains a bunch of unnecessary code reformatting. Can you only include the code that's actually changing, and match the style of the existing code?

Removed the formatting, and changed the formatting of the new code that now matches the style i hope

samskivert commented:
Thanks. It looks pretty good. Do you mind squashing the commits in the PR to just one commit that implements the change? I prefer not to junk up the commit history with a bunch of review commits. It makes it harder to see what changed when and why.

pweibert commented:
> Thanks. It looks pretty good. Do you mind squashing the commits in the PR to just one commit that implements the change? I prefer not to junk up the commit history with a bunch of review commits. It makes it harder to see what changed when and why.
I also think squashing is the best way to go +1:

pweibert commented:
Sorry, but I do not see how I can squash it without write permissions over the github interface or shoud squash locally and then force push?

samskivert commented:
Right, squash locally, force push on your branch and then the PR will only
contain a single commit.

On Fri, Nov 22, 2019 at 2:31 PM pweibert <notifications@github.com> wrote:

> Sorry, but I do not see how I can squash it without write permissions over
> the github interface or shoud squash locally and then force push?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/threerings/tripleplay/pull/78?email_source=notifications&email_token=AABP55B6OIHM7W2MZ7NFPB3QVBMUTA5CNFSM4JHTVDH2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEE7BZKI#issuecomment-557718697>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABP55CDQNXF6VCJCPZCIKDQVBMUTANCNFSM4JHTVDHQ>
> .
>


-- 
-- mdb@samskivert.com

pweibert commented:
> Right, squash locally, force push on your branch and then the PR will only contain a single commit.
Squashed!

samskivert commented:
Thanks! Sorry for the delay in merging. Busy busy.
